### DISCUSSION TITLE
AWS data access

### DISCUSSION THREAD

**Discussion by s-canchi** [nih-cfde/training-and-engagement#411] [AWS]
If you have data in an AWS S2 bucket, how can you access it from EC2 clusters?

**Comment by mr-eyes** [comment]
https://aws.amazon.com/premiumsupport/knowledge-center/ec2-instance-access-s3-bucket/

**Comment by ctb** [comment]
You might be able to mount it with a S3/FUSE system. But most likely you want to just download it directly, since block device/file system access to it is not super efficient.  s3cmd offers some nice options for downloading.

EBS / block devices are how you can do random access a la your typical file system approach.
